src/lib/auth.ts

```
@@ -0,0 +1,31 @@
+import { auth } from "@clerk/nextjs/server";
+import { redirect } from "next/navigation";
+import type { SessionMetadata, SessionClaims, AuthSession } from "../types/auth";
```

ESLint

[error] 'AuthSession' is defined but never used.

(@typescript-eslint/no-unused-vars)